karma.conf.js

```diff
@@ -25,7 +25,7 @@ function parseArguments () {
     // Last argument is the browser
     let browser = process.argv[process.argv.length - 1]
     if (!browser || browser.includes('karma') || browser === 'start') {
-      browser = 'phantomjs'
+      browser = 'chromeHeadless'
```

Comment:
can we just have this be `chrome` instead of `chromeHeadless`?

Reply:
Sadly we can't because chrome it's the identifier for SauceLab's Chrome :S